src/arviz_plots/plots/distplot.py

```diff
@@ -250,6 +253,16 @@ def plot_dist(
                 **density_kwargs,
             )
 
+        elif kind == "hist":
+            density = distribution.azstats.histogram(
+                dims=density_dims, density=True, **stats_kwargs.get("density", {})
```

Comment:
`density=True` should be added to `stats_kwargs` with `setdefault`. Then, if a user wanted to they can set it to False, keep the axis and use the y axis to look at counts for example. The task itself isn't really important, what is is that it should be possible